lib/optimize.js

```diff
@@ -1,25 +1,8 @@
-const imagemin = require('imagemin');
-const imageminJpegtran = require('imagemin-jpegtran');
-const imageminPngquant = require('imagemin-pngquant');
-const imageminSvgo = require('imagemin-svgo');
-const imageminWebp = require('imagemin-webp');
+const sharp = require('sharp');
 
-module.exports = (options, buffer) => {
-  const plugins = [];
-  if (options.svgo !== false) {
-    plugins.push(imageminSvgo(options));
-  }
-  if (options.png !== false) {
-    plugins.push(imageminPngquant(options));
-  }
-  if (options.jpg !== false) {
-    plugins.push(imageminJpegtran());
-  }
-  if (options.webp !== false) {
-    plugins.push(imageminWebp());
-  }
+module.exports = async(buffer) => {
   try {
-    return imagemin.buffer(buffer, { plugins });
+    return sharp(buffer).toBuffer();
   } catch (err) {
     // anything other than error 99 is considered an unrecoverable error:
     if (err.code !== 99) {
```

Comment:
Does this err code still apply? What does it do? We should make sure this block still works.

Reply:
Just so I'm clear ... do we know what happens when sharp throws an error? Does their error object have a code? If all errors are un recoverable, shouldn't we just remove the try / catch block and let all errors throw? It seems to me like we should match this code with the library, and not leave snippets lying around that don't have meaning.